docs/getting-started/integration.md

```diff
@@ -15,87 +15,82 @@ Chart.js can be integrated with plain JavaScript or with different module loader
 
 Chart.js 3 is tree-shakeable, so it is necessary to import and register the controllers, elements, scales and plugins you are going to use.
 
-For all available imports see the example below.
+### Quick start
+
+If you don't care about the bundle size, you can use the `auto` package ensuring all features are available:
 
 ```javascript
-import {
-  Chart,
-  ArcElement,
-  LineElement,
-  BarElement,
-  PointElement,
-  BarController,
-  BubbleController,
-  DoughnutController,
-  LineController,
-  PieController,
-  PolarAreaController,
-  RadarController,
-  ScatterController,
-  CategoryScale,
-  LinearScale,
-  LogarithmicScale,
-  RadialLinearScale,
-  TimeScale,
-  TimeSeriesScale,
-  Decimation,
-  Filler,
-  Legend,
-  Title,
-  Tooltip,
-  SubTitle
-} from 'chart.js';
-
-Chart.register(
-  ArcElement,
-  LineElement,
-  BarElement,
-  PointElement,
-  BarController,
-  BubbleController,
-  DoughnutController,
-  LineController,
-  PieController,
-  PolarAreaController,
-  RadarController,
-  ScatterController,
-  CategoryScale,
-  LinearScale,
-  LogarithmicScale,
-  RadialLinearScale,
-  TimeScale,
-  TimeSeriesScale,
-  Decimation,
-  Filler,
-  Legend,
-  Title,
-  Tooltip,
-  SubTitle
-);
-
-const myChart = new Chart(ctx, {...});
+import Chart from 'chart.js/auto';
 ```
 
-A short registration format is also available to quickly register everything.
+Or if you are planning to optimize the bundle later, you can start by importing and registering `registerables`, which also ensures all features are available:
 
 ```javascript
 import { Chart, registerables } from 'chart.js';
 Chart.register(...registerables);
 ```
 
-And finally there is a separate path to do just the above for you, in one line:
-
-```javascript
-import Chart from 'chart.js/auto';
-```
-
-## CommonJS
-
-Because Chart.js is an ESM library, in CommonJS modules you should use a dynamic `import`:
-
-```javascript
-const { Chart } = await import('chart.js');
-```
+### Bundle optimization
+
+When optimizing the bundle, instead of the `registerables`, you need to import and register the components that are needed in your application.
+
+The options are categorised (inspect `registerables`) into controllers, elements, plugins, scales. You can pick and choose many of these, e.g. if you are not going to use tooltips, don't import and register the `Tooltip` plugin. But each type of chart has its own bare-minimum requirements (typically the type's controller, element(s) used by that controller and scale(s)):
+
+* Bar chart
+  * `BarController`
+  * `BarElement`
+  * Default scales: x: `CategoryScale`, y: `LinearScale`
```

Comment:
it's a little weird to have multiple `:`. I would change one to a `-`. (and below as well)
```suggestion
  * Default scales - x: `CategoryScale`, y: `LinearScale`
```